### DISCUSSION TITLE
readme.md

### DISCUSSION THREAD

**Discussion by KeremBalci111** [irgroup-classrooms/dis08-2021#73] [Q&A]
Guten Abend Herr Schaer,

Wir haben unseren Code bereits zu genüge im Jupyter Notebook kommentiert und erklärt, ist es dennoch notwendig eine "complementing README.md" zu verfassen wie sie es in den Rules gefordert haben?

Beste Grüße,
Kerem Balci

**Comment by phschaer** [comment]
Nein, wenn alles im Notebook dokumentiert ist, ist das in Ordnung.
